ares.h

```diff
@@ -339,6 +339,8 @@ CARES_EXTERN const char *ares_version(int *version);
 
 CARES_EXTERN int ares_init(ares_channel *channelptr);
 
+CARES_EXTERN int ares_reinit_by_resolv_conf_file(ares_channel channel);
```

Comment:
I think this should be just "ares_reinit"

- not all systems may have resolv_conf so implying that in the symbol name seems wrong
- not all systems may reinit from a"file" so lets not say so either

This then allows other systems to do whatever is necessary there to do the equivalent.

Reply:
Thanks for feedback.
Maybe the function has wrong name and is on wrong place.

I just wanted a function that would internally reload solely the resolv.conf file if it is used by that particular system. It probably does not belong to ares.h - I just wanted to access it easily from ares_gethostbyname.c.

But maybe other systems like Windows can change DNS information on the fly (in registers?) or something and it would have the same problem. 
But I do not own Windows or other platforms, just Linux.

So I can probably rename the function to ares_reload_resolv_conf() and move it to non public place (but where?). And then what do I do with other platform implementations?
I could make some effort and try to implement things blindly, not sure if its good idea.